website/docs/docs/dbt-versions/release-notes/73-Jan-2024/jan-sl-updates.md

```diff
@@ -0,0 +1,36 @@
+
+## Updates
+
+- **Display `label` for dbt Semantic Layer metrics** &mdash; The YAML spec parameter `label` is now available for Semantic Layer metrics in [JDBC and GraphQL APIs](/docs/dbt-cloud-apis/sl-api-overview). This means you can conveniently use `label` as a display name for your metrics when exposing them.
+- **Use shorthand to create metrics** &mdash; Added support for `create_metric: true` for a measure, which is a shorthand to quickly create metrics. This is useful in cases when metrics are only used to build other metrics.
+- **Support for Tableau parameter filters** &mdash; Added support for Tableau parameter filters. You can use the [Tableau connector](docs/use-dbt-semantic-layer/tableau) to create and use parameters with your dbt Semantic Layer data.
+- **Additional parameters for GraphQL API** &mdash;  Added support to expose `expr` and `agg` for [Measures](/docs/build/measures) in the [GraphQL API](/docs/dbt-cloud-apis/sl-graphql).
+- **Improved error messages** &mdash; You have improved error messages in the command line interface when querying a dimension that is not reachable for a given metric.
+- **Entities in Tableau** &mdash; You can now query entities using our Tableau integration (similar to querying dimensions). 
+- **New Tableau data source** &mdash; A new data source is available in our Tableau integration called "ALL", which contains all semantic objects defined. This has the same information as "METRICS_AND_DIMENSIONS". In the future, we will deprecate "METRICS_AND_DIMENSIONS" in favor of "ALL" for clarity. 
+
+## Bug fixes
+
+- **BigQuery numeric types** &mdash; Support for numeric types with precision greater than 38 (like `BIGDECIMAL`) in BigQuery is now available. Previously, it was unsupported so would return an error.
+- **Support for scientific notation** &mdash; Scientific notation for large numbers is displayed and correctly interpreted. 
+- **Google Sheets dimension values** &mdash; We now preserve dimension values accurately instead of being inadvertently converted into strings. 
+- **Multiple derived metrics resolution** &mdash; Resolved issues with naming collisions in queries involving multiple derived metrics using the same metric input. Previously, this could cause a naming collision. Input metrics are now deduplicated, ensuring each is referenced only once.
+- **Deduplication of input measures** &mdash; Resolved warnings related to using two duplicate input measures in a derived metric. Previously, this would trigger a warning. Input measures are now deduplicated, enhancing query processing and clarity.
+- **Correctly reference entities in filters using the object syntax** &mdash; Resolved an error where referencing an entity in a filter using the object syntax would fail. i.e {{Entity('entity_name')}} would fail to resolve.
```

Comment:
```suggestion
- **Correctly reference entities in filters using the object syntax** &mdash; Resolved an error where referencing an entity in a filter using the object syntax would fail. For example, `{{Entity('entity_name')}}` would fail to resolve.
```